frontend/packages/dev-console/src/components/topology/list-view/cells/MetricsCell.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Node } from '@patternfly/react-topology';
+import { DataListCell, Tooltip } from '@patternfly/react-core';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { formatBytesAsMiB, formatCores, truncateMiddle } from '@console/internal/components/utils';
+import { useOverviewMetrics } from '../../../../utils/useOverviewMetrics';
+import { isMobile } from '../list-view-utils';
+
+type MetricsProps = {
+  item: Node;
+};
+
+type MetricsTooltipProps = {
+  metricLabel: string;
+  byPod: {
+    formattedValue: string;
+    name: string;
+    value: number;
+  }[];
+};
+
+const MetricsTooltip: React.FC<MetricsTooltipProps> = ({ metricLabel, byPod, children }) => {
+  const sortedMetrics = _.orderBy(byPod, ['value', 'name'], ['desc', 'asc']);
+  const content: any[] = _.isEmpty(sortedMetrics)
+    ? [<React.Fragment key="no-metrics">No {metricLabel} metrics available.</React.Fragment>]
+    : _.concat(
+        <div className="project-overview__metric-tooltip-title" key="#title">
+          {metricLabel} Usage by Pod
+        </div>,
+        sortedMetrics.map(({ name, formattedValue }) => (
+          <div key={name} className="project-overview__metric-tooltip">
+            <div className="project-overview__metric-tooltip-name">
+              <span className="no-wrap">{truncateMiddle(name)}</span>
+            </div>
+            <div className="project-overview__metric-tooltip-value">{formattedValue}</div>
+          </div>
+        )),
+      );
+
+  const keepLines = 6;
+  // Don't remove a single line to show a "1 other" message since there's space to show the last pod in that case.
+  // Make sure we always remove at least 2 lines if we truncate.
+  if (content.length > keepLines + 1) {
+    const numRemoved = content.length - keepLines;
+    content.splice(
+      keepLines,
+      numRemoved,
+      <div key="#removed-pods">and {numRemoved} other pods</div>,
+    );
+  }
+
+  // Disable the tooltip on mobile since a touch also opens the sidebar, which
+  // immediately covers the tooltip content.
+  if (isMobile()) {
+    return <>{children}</>;
+  }
+  return (
+    <Tooltip content={content} distance={15}>
+      <>{children}</>
```

Comment:
```suggestion
      {children}
```
Don't think you need the fragment inside the tooltip.

Reply:
Need this due to Tooltip requiring a single element.